src/luacheck/builtin_standards/init.lua

```diff
@@ -31,6 +31,7 @@ string_defs.lua51 = add_defs(string_defs.min, standards.def_fields("gfind"))
 string_defs.lua52 = string_defs.min
 string_defs.lua53 = add_defs(string_defs.min, standards.def_fields("pack", "packsize", "unpack"))
 string_defs.lua54 = string_defs.lua53
+string_defs.lua55 = string_defs.lua53
```

Comment:
Shouldn't the 5.5 presets build on 5.4 ... and do so based on a copy after the 5.4 modifications from 5.3 have been applied?

Reply:
Fair enough. I realized that might be the case after posting. I suppose it doesn't make much difference in the end, it just saves knowing that bit of trivia when reading the code.